Cogs/misc.py

```diff
@@ -18,8 +18,8 @@ async def help(self, ctx, type = None):
             embed.set_author(name = "Type `!help [category]` to see an in-depth guide for the inputed category")
             embed.set_footer(text = "Created by DMLooter#4251 & PureCache#0001")
             embed.add_field(name = "Verification Commands", value = data["Verification"], inline = False)
+            embed.add_field(name = "Website Commands", value = data["Website"], inline = False)
             embed.add_field(name = "Miscellaneous Commands", value = data["Misc"], inline = False)
-            #embed.add_field(name = "Settings Commands", value = data["Settings"], inline = False)

```

Comment:
Is there a reason settings got commented out?

Reply:
Not that I can remember. I might say it was because it was showing up to people who cant use it (and we basically don't use those commands anyways)

Reply:
We can comment it out for now, i'm gonna make an issue for a pr that adds permission checks to the necessary help commands